packages/editor/src/client/panel/embedEditPanel.tsx

```diff
@@ -66,17 +65,37 @@ export function EmbedEditPanel({value, onClose, onConfirm}: EmbedEditPanel) {
 
         if (iframe) {
           const soundCloudMatch = iframe.src.match(/api.soundcloud.com\/tracks\/([0-9]+)/)
+          const ratioVal = iframe.hasAttribute('style')
```

Comment:
I don't understand this and the following code. Maybe we should look at this together.

Reply:
it basically just checks if `style` attribute is in place or not ... if yes the state mentioned above is set to `false`

Reply:
is removed and more readable and explainable implemented